scripts/load_submissions_graph.py

```diff
@@ -0,0 +1,112 @@
+import argparse
+import logging
+import os
+import subprocess
+import sys
+import time
+from typing import List
+
+import pandas as pd
+
+SOLUTIONS_DIR_NAME = "solutions"
+OUTPUT_DIR_NAME = "output"
+
+
+def build_solutions_file_name(step_id: int, output_path: str) -> str:
+    return f"{output_path}/{SOLUTIONS_DIR_NAME}/{step_id}.csv"
+
+
+def build_output_dir_name(step_id: int, output_path: str) -> str:
+    return f"{output_path}/{OUTPUT_DIR_NAME}/{step_id}"
+
+
+def configure_load_cmd(input_file: str, output_dir: str, serialize: bool, save_csv: bool) -> str:
+    cmd = f"./gradlew :code-submissions-clustering-plugin:load -Pinput={input_file} -Poutput={output_dir}"
+    if serialize:
+        cmd = cmd + " -Pserialize"
+    if save_csv:
+        cmd = cmd + " -PsaveCSV"
+    return cmd
+
+
+def parse_solutions(input_file: str, output_path: str) -> List[int]:
+    df_all_solutions = pd.read_csv(input_file)
+    unique_steps = df_all_solutions['step_id'].unique()
+    for step in unique_steps:
+        output_file = build_solutions_file_name(step, output_path)
+        df_all_solutions[df_all_solutions['step_id'] == step].to_csv(output_file, index=False)
+    return unique_steps
+
+
+def run_submissions_graph_load(step_id: int, output_path: str, serialize: bool, save_csv: bool):
+    input_file = build_solutions_file_name(step_id, output_path)
+    output_dir = build_output_dir_name(step_id, output_path)
+    cmd = configure_load_cmd(input_file, output_dir, serialize, save_csv)
+    subprocess.run(cmd.split(), stdout=subprocess.DEVNULL, stderr=subprocess.DEVNULL)
+
+
+def set_logger(output_path: str) -> logging.Logger:
```

Comment:
It should be moved into utilities, because it will be reused in other scripts in the future